src/components/sbb-card-product/sbb-card-product.stories.js

```diff
@@ -237,7 +237,11 @@ const sbbTitleTravelCardHalfFareArgs = {
   'visual-level': 1,
 };
 
-const SlotSbbTitleTemplate = (args) => <sbb-title {...args} />;
+const SlotSbbTitleTemplate = (args) => (
+  <sbb-title {...args}>
+    <span slot="content">{args.text}</span>
```

Comment:
Please adapt all usages to directly slot the content without the span. See comment about nameless slot.